test/unittests/commonMocks.h

```diff
@@ -160,13 +161,14 @@ class NotifierMock : public Notifier {
          * actually created/sent */
     }
 
-    MOCK_METHOD6(sendNotifyContextRequest, void(NotifyContextRequest* ncr, const std::string& url, const std::string& tenant, const std::string& xauthToken, const std::string& fiwareCorrelator, Format f));
+    MOCK_METHOD6(sendNotifyContextRequest, void(NotifyContextRequest* ncr, const std::string& url, const std::string& tenant, const std::string& xauthToken, const std::string& fiwareCorrelator, NotificationFormat notifyFormat));
     MOCK_METHOD5(sendNotifyContextAvailabilityRequest, void(NotifyContextAvailabilityRequest* ncar, const std::string& url, const std::string& tenant, const std::string& fiwareCorrelator, Format f));
 
     /* Wrappers for parent methods (used in ON_CALL() defaults set in the constructor) */
-    void parent_sendNotifyContextRequest(NotifyContextRequest* ncr, const std::string& url, const std::string& tenant, const std::string& xauthToken, const std::string& fiwareCorrelator, Format format)
+    void parent_sendNotifyContextRequest(NotifyContextRequest* ncr, const std::string& url, const std::string& tenant, const std::string& xauthToken, const std::string& fiwareCorrelator, NotificationFormat notifyFormat)
     {
-      return Notifier::sendNotifyContextRequest(ncr, url, tenant, xauthToken, fiwareCorrelator, format);
+      LM_W(("KZ: In parent_sendNotifyContextRequest"));
```

Comment:
Yeah, I did one, as I always do.
But of course, I look in *.cpp, not in *.h ...
This is the only one